cypress/support/fragments/marcAuthority/marcAuthorities.js

```diff
@@ -388,6 +388,11 @@ export default {
     cy.do(Checkbox(value).click());
   },
 
+  downloadSelectedRecordWithRowIdx(checkBoxNumber = 1) {
+    cy.get(`div[class^="mclRow--"]:nth-child(${checkBoxNumber}) input[type="checkbox"]`).click();
+    cy.do([actionsButton.click(), Button('Export selected records (CSV/MARC)').click()]);
```

Comment:
Move interactor  Button('Export selected records (CSV/MARC)') to const variable please.